cmd/dagger/licenses.go

```diff
@@ -62,19 +61,19 @@ var licenseFiles = []string{
 }
 
 func findOrCreateLicense(ctx context.Context, dir string) error {
-	log := telemetry.ContextLogger(ctx, slog.LevelWarn)
```

Comment:
`ContextLogger` and `ioctx` are gone now

Reply:
:pray: thank goodness :smile:

Genuinely happy to see these gone, they were quite confusing :)